contracts/src/v0.8/functions/dev/1_0_0/FunctionsClient.sol

```diff
@@ -24,29 +22,11 @@ abstract contract FunctionsClient is IFunctionsClient {
     i_router = IFunctionsRouter(router);
   }
 
-  /**
-   * @notice Sends a Chainlink Functions request
-   * @param request The Functions.Request initialized in memory
-   * @param subscriptionId The subscription ID that will be charged to service the request
-   * @param callbackGasLimit the amount of gas that will be available for the fulfillment callback
-   * @return requestId The generated request ID for this request
-   */
-  function _sendRequest(
```

Comment:
Why'd we remove this? Any objection to simplifying it so the one method is called `_sendRequest`? Shorter, simpler, and it matches what it calls on the router.

Reply:
No objections, changing

Reply:
> Why'd we remove this?
With the feature that Morgan is working on for signed request CBOR, it won't be useful anymore, so just cleaning it out.